.github/workflows/artifacts.yml

```diff
@@ -3,28 +3,28 @@ on:
   workflow_dispatch:
     inputs:
       core:
-        description: "Version of the core. Must go with v (e.g v0.2.47)"
+        description: "Version of the Core. Must be prefixed with v (e.g v0.2.47)"
```

Comment:
I am also often guily of this, but we should keep our PRs as single issue as we can. Preferably, this should be in a separate PR which we can merge without going through all the testing necessary for the other stuff. 

Keep it as it is now, this comment is just something to consider for the future.

Reply:
yeah i just did it because i had to update the workflows to make it create ISOs with Bookworm in the name not bullseye, so i just slipped it in because the typos had been driving me nuts.  I try to keep things separate but just had to do something since i already had to edit the workflows for this branch anyway., but yeah i do it too often mixing in extra changes in PRs, ill try and keep to proper etiquette in my PRs and commits